style.css

```
@@ -15,6 +15,37 @@ a {
     text-decoration: none;
 }
 
+.main-header {
+    background-color: #eeeeee;
+    padding-bottom: 30px;
+}
+
+.main-navigation {
+    padding-top: 45px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.container {
+    width: 1160px;
```

```suggestion
    width: 1200px;
    padding-left: 20px;
    padding-right: 20px;
```
чтоб быть уверенным что контент не прижмется к краю экрана вплотную.